### ISSUE TITLE
Attachments are not displayed in an attached *.EML file

### ISSUE THREAD

**Issue by EtagiBI** [newroco/emlviewer#61]
Steps to reproduce:
1. Send any email with some attachments to your **non-Nextcloud** email address
2. Log in to your **non-Nextcloud** email address
3. Resend this received email as attachment to your **Nextcloud** email address
4.  Log in to your **Nextcloud** email address
5. Open *.EML file from the received email
6. The problem is attachemnts are not displayed within opened *.EML file

**Comment by putt1ck** [comment]
To check: if the same email forwarded in the normal way rather than as an attachment, the attachments of the forwarded email open as expected?

**Comment by EtagiBI** [comment]
> To check: if the same email forwarded in the normal way rather than as an attachment, the attachments of the forwarded email open as expected?

Yes, the problem affects only attached emails in a printer-friendly view
